packages/ozone/src/api/moderation/emitEvent.ts

```diff
@@ -165,13 +166,20 @@ const handleModerationEvent = async ({
       throw new InvalidRequestError('Email can only be sent to a repo subject')
     }
     const { content, subjectLine } = event
-    await retryHttp(() =>
-      ctx.modService(db).sendEmail({
-        subject: subjectLine,
-        content,
-        recipientDid: subject.did,
-      }),
-    )
+    // on error, don't fail the whole event. instead, log the event data with isDelivered false
+    try {
+      await retryHttp(() =>
+        ctx.modService(db).sendEmail({
+          subject: subjectLine,
+          content,
+          recipientDid: subject.did,
+        }),
+      )
+      event.isDelivered = true
+    } catch (err) {
+      httpLogger.error({ err, event }, 'failed to send mod event email')
+      event.isDelivered = false
```

Comment:
To match the comment on line 169 it'd be:
```suggestion
      event.isDelivered = false
      httpLogger.error({ err, event }, 'failed to send mod event email')
```